app/src/features/error-boundary/GenerellFeilside.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  BodyShort,
+  Box,
+  Button,
+  Heading,
+  HGrid,
+  Link,
+  List,
+  Page,
+  VStack,
+} from "@navikt/ds-react";
+import { PageBlock } from "@navikt/ds-react/Page";
+
+export const GenerellFeilside = () => {
+  return (
+    <Page>
+      <PageBlock as="main" gutters width="xl">
+        <Box paddingBlock="20 8">
+          <HGrid columns="minmax(auto,600px)" data-aksel-template="500-v2">
+            <VStack gap="16">
+              <VStack align="start" gap="12">
+                <div>
+                  <BodyShort size="small" textColor="subtle">
+                    Statuskode 500
```

Comment:
Trenger vi/burde vi si noe om statuskode? For brukere gir det ingenting, og for oss er det for generelt til å hjelpe oss også 🤷 stemmer for å droppe

Reply:
Fair nok! Kan fjerne den jeg :)